addons/crm/data/crm_lead_prediction_data.xml

```diff
@@ -23,6 +23,10 @@
         <record id="frequency_field_tag_ids" model="crm.lead.scoring.frequency.field">
             <field name="field_id" ref="crm.field_crm_lead__tag_ids"/>
         </record>
+        <record id="crm_iap_lead_enrich_setting" model="ir.config_parameter">
+            <field name="key">crm.iap.lead.enrich.setting</field>
+            <field name="value">auto</field>
+        </record>
```

Comment:
Instead we could remove this data and consider the default to be "auto" when calling "get_param".

So instead of 
`enrich_mode = self.env['ir.config_parameter'].sudo().get_param('crm.iap.lead.enrich.setting', 'manual')`

You would have
`enrich_mode = self.env['ir.config_parameter'].sudo().get_param('crm.iap.lead.enrich.setting', 'auto')`

Isn't it more simple?